pep-0639.rst

```diff
@@ -1,543 +1,2221 @@
 PEP: 639
-Title: Metadata for Python Software Packages 2.2
+Title: Improving License Clarity with Better Package Metadata
 Version: $Revision$
 Last-Modified: $Date$
-Author: Philippe Ombredanne <pombredanne at nexb.com>
+Author: Philippe Ombredanne <pombredanne at nexb.com>,
+        C.A.M. Gerlach <CAM.Gerlach at Gerlach.CAM>
 Sponsor: Paul Moore <p.f.moore at gmail.com>
-BDFL-Delegate: Paul Moore <p.f.moore at gmail.com>
+PEP-Delegate: Paul Moore <p.f.moore at gmail.com>
 Discussions-To: https://discuss.python.org/t/2154
 Status: Draft
 Type: Standards Track
 Content-Type: text/x-rst
 Created: 15-Aug-2019
-Python-Version: 3.x
 Post-History:
-Replaces: 566
 Resolution:
 
 
 Abstract
 ========
 
-This PEP describes the changes between versions 2.1 and 2.2 of the `Core
-Metadata Specification` [#cms]_ for Python packages. Version 2.1 is specified in
-PEP 566.
+This PEP defines a specification for how licenses are documented in the
+`core metadata <#coremetadataspec_>`_,
```

Comment:
I haven’t see trailing underscore inside the angle brackets like this — any reason?

Reply:
Yep, its required in both Sphinx and docutils for indirect hyperlink references to both sections and external URIs per the [reST spec](https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#embedded-uris-and-aliases), otherwise they simply don't work at all.

At least for internal section links in Sphinx, I'd normally use the `:ref:` role (which doesn't require this in `<>`) and explicitly named targets to avoid breaking inbound links when changing section names, but that of course isn't supported in plain docutils, compatibility which we have to retain until [PEP 676](https://www.python.org/dev/peps/pep-0676/) is accepted. For hyperlinks, the extra layer of indirection is needed due to minimize tedious refactoring while implementing #2130 and retain most of the existing structure, as well as due to the large number (>100) total link references and multiple references for many links while maintaining unique, robust and concise target names.